dist/index.js

```diff
@@ -2611,9 +2611,9 @@ function parse(options) {
 
   const omittedParameters = Object.keys(options).filter(option => urlVariableNames.includes(option)).concat("baseUrl");
   const remainingParameters = omit(parameters, omittedParameters);
-  const isBinaryRequset = /application\/octet-stream/i.test(headers.accept);
+  const isBinaryRequest = /application\/octet-stream/i.test(headers.accept);
```

Comment:
For my future reference:
https://github.com/octokit/endpoint.js/blob/dc03f51dbf580ba9f3a771346a3650ee8809e787/src/parse.ts#L43